src/azureFunction/azureFunctionProjectService.ts

```diff
@@ -0,0 +1,175 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as os from 'os';
+import * as vscode from 'vscode';
+import * as af from '../../typings/vscode-azurefunctions.api';
+import * as cp from 'child_process';
+import * as fs from 'fs';
+import * as mssql from 'vscode-mssql';
+import { azureFunctionsExtensionName, defaultBindingResult, defaultSqlBindingTextLines, genericCollectionImport, sqlBindingResult, sqlConnectionString } from '../constants/constants';
+import LocalizedConstants = require('../constants/localizedConstants');
+import path = require('path');
+import { AzureFunctionsService } from '../services/azureFunctionsService';
+
+const sqlBindingNugetSource = 'https://www.myget.org/F/azure-appservice/api/v3/index.json';
+const sqlBindingPackageName = 'Microsoft.Azure.WebJobs.Extensions.Sql';
+const sqlBindingPackageVersion = '1.0.0-preview3';
```

Comment:
it would be nice if it pulls in the latest version so this doesn't need to be updated every time a new version is released